### PR TITLE
Feature/hana installation

### PR DESCRIPTION
**Pull request by arbulu89**
Updates for the task:
https://trello.com/c/fp52Pvf6/227-create-salt-formula-for-sap-hana-system-replication-and-installation-modules

Add HANA installation features (modules and states)

### CODE REVIEW

File: salt/__init__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+"""
+:author: xarbulu
+:organization: SUSE Linux GmbH
+:contact: xarbulu@suse.com
+
+:since: 2018-11-30
+"""
+
+__version__ = "0.2.0"
```
Comment (diegoakechi) [inline salt/__init__.py]: Maybe to be coherent with shaptools, we should only increase this version number after the first release...

Comment (arbulu89) [reply salt/__init__.py]: Agree!

**Comment by arbulu89** [comment]
@diegoakechi Could you accept the PR

File: salt/modules/hanamod.py [hunk 2]
```diff
@@ -98,7 +198,7 @@ def is_running(sid=None, inst=None, password=None):
 
     .. code-block:: bash
 
-        salt '*' hana.is_running prd 00 pass
+        salt '*' hana.is_running prd '"00"' pass
```
[the remark this answers is not in the document]
Comment (arbulu89) [reply salt/modules/hanamod.py]: Yes, this is due yaml parsing, it does strange things. I don't really like even how it works, but by now is better to follow the salt recommendations, and they use this syntax.

**Pull request merged**